apps/estuary/config/prod.exs

```diff
@@ -0,0 +1,4 @@
+use Mix.Config
+
+# Do not print debug messages in production
```

Comment:
I appreciate the comments here, but they're probably unnecessary.

Reply:
Agreed